### ISSUE TITLE
How can a resizing ruin your normalization

### ISSUE THREAD

**Issue by Data-Iab** [Visual-Behavior/aloception#197]
```python

frame = torch.ones((3, 50, 50))
frame[0, 0, 0 : 10] = 0

frame = aloscene.Frame(frame, normalization="01", names=tuple("CHW"))
frame = frame.norm_minmax_sym()

print("(min, max) :", frame.min().item(), frame.max().item())
frame = frame.resize((20, 20)) 

frame = frame.norm_minmax_sym()  # nothing changes, just a clone
print("(min, max) :", frame.min().item(), frame.max().item())
```

```python
(min, max) : -1.0 1.0
(min, max) : 0.5 1.0

```

**Comment by thibo73800** [comment]
If you had a real issue I'm not seeing the issue with the provided code

**Comment by Data-Iab** [comment]
@thibo73800
This code will make things clearer:

```python
frame = torch.rand((3, 50, 50))

frame = aloscene.Frame(frame, normalization="01", names=tuple("CHW"))
frame = frame.norm_minmax_sym()

print("(min, max) :", frame.min().item(), frame.max().item())
rframe = frame.resize((40, 40))

rframe = rframe.norm_minmax_sym()
print("(min, max) :", rframe.min().item(), rframe.max().item())

```

```python
(min, max) : -0.9994168281555176 0.9998884201049805
(min, max) : -0.9518397450447083 0.9138493537902832

```
There are two issues:
- The normalization after resizing does not change anything, if at some point (after defining a data iterator that already normalizes) you want to resize and normalize again, it's not gonna work and you'll end up having a range of values different than what you expect (I suppose using this frame with a model trained with [-1, 1] range of values will end up with a different output).
- Some transforms use this method (RandomResizeWithAspectRatio, Resize, RandomDownScale, RandomDownScale). Any training pipeline with those trans will not input a constant value range.

--------------------------------
Except for the NEAREST interpolation, the value range is not kept the same
```python
# InterpolationMode.BICUBIC
(min, max) : -0.9998984336853027 0.9995183944702148
(min, max) : -1.2766730785369873 1.2466816902160645

# InterpolationMode.NEAREST
(min, max) : -0.9988489151000977 0.9999920129776001
(min, max) : -0.9988489151000977 0.9999920129776001

```

**Comment by thibo73800** [comment]
I think this behavior is due to the interpolation mode but not directly to ``aloscene`
The following show the behavior both through tensor only and using aug tensor

Ouput are the same

```python
import torch
import aloscene
import torch.nn.functional as F
from torchvision.transforms import InterpolationMode


torch.manual_seed(42)
frame = torch.rand((1, 3, 50, 50))

frame = aloscene.Frame(frame, normalization="01", names=tuple("BCHW"))
frame = frame.norm_minmax_sym()

# Bilinear
wo_aug_tensor = F.interpolate(frame.as_tensor(), size=(40, 40), mode="bilinear", align_corners=False) 
w_aug_tensor = frame.resize((40, 40))
print("wo_aug_tensor (min, max) :", wo_aug_tensor.shape, wo_aug_tensor.min().item(), wo_aug_tensor.max().item())
print("w_aug_tensor (min, max) :", w_aug_tensor.shape, w_aug_tensor.min().item(), w_aug_tensor.max().item())

assert torch.allclose(wo_aug_tensor, w_aug_tensor)

# Nearest
wo_aug_tensor = F.interpolate(frame.as_tensor(), size=(40, 40), mode="nearest") 
w_aug_tensor = frame.resize((40, 40), interpolation=InterpolationMode.NEAREST)
print("wo_aug_tensor (min, max) :", wo_aug_tensor.shape, wo_aug_tensor.min().item(), wo_aug_tensor.max().item())
print("w_aug_tensor (min, max) :", w_aug_tensor.shape, w_aug_tensor.min().item(), w_aug_tensor.max().item())

assert torch.allclose(wo_aug_tensor, w_aug_tensor)
```
